controller/work_item_labels_blackbox_test.go

```diff
@@ -37,6 +37,7 @@ func TestRunWorkItemLabelREST(t *testing.T) {
 }
 
 func (l *TestWorkItemLabelREST) SetupTest() {
+	l.DBTestSuite.SetupTest()
 	l.db = gormapplication.NewGormDB(l.DB)
```

Comment:
Overall looks good. But we did some improvements in Auth service to the DBTestSuite which I think WIT would benefit too - https://github.com/fabric8-services/fabric8-auth/blob/master/gormtestsupport/db_test_suite.go
For example there is no need actually to create Gorm Application in ever suite implementation (should be moved to the DBTest suite). We also improved the cleaner functions, etc.

Hope we will start moving the common code from our services to upstream and re-use it everywhere at some point. Instead of keep re-inviting it for every service ;)

Reply:
@alexeykazakov be my guest to make a standalone package out of your test improvements. We're happy to contribute and consume that.